fulfillment-sdk/bin/pom.xml

```diff
@@ -0,0 +1,206 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.kibocommerce</groupId>
+    <artifactId>inventory-sdk</artifactId>
+    <packaging>jar</packaging>
+    <name>inventory-sdk</name>
```

Comment:
fulfillment

Reply:
actually don't know where the bin directory came from. If I delete it and then regenerate it is not regenerated. I think this was left over from a copy and paste of the inventory sdk.